os/kernel/task/task_prctl.c

```diff
@@ -166,6 +166,16 @@ int prctl(int option, ...)
 	goto errout;
 #endif
 
+	case PR_COPY_STKMONLOG:
+	{
+#if defined(CONFIG_ENABLE_STACKMONITOR) && defined(CONFIG_DEBUG)
+		struct stkmon_save_s *dest_arr = va_arg(ap, struct stkmon_save_s *);
+		stkmon_copy_log(dest_arr);
+#else
+		sdbg("Not supported StackMonitor Logging\n");
```

Comment:
err = EINVAL;

Reply:
I think ENOSYS is better, so add `err = ENOSYS`.